docs/src/main/asciidoc/stork-kubernetes.adoc

```diff
@@ -0,0 +1,479 @@
+
+
+== The REST Client interface and the front end API
+
+So far, we didn't use Stork; we just deployed the services we will be discovering, selecting, and calling.
+
+We will call the services using the Reactive REST Client.
+Create the `src/main/java/org/acme/MyService.java` file with the following content:
+
+[source, java]
+----
+package org.acme;
+
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+/**
+ * The REST Client interface.
+ *
+ * Notice the `baseUri`. It uses `stork://` as URL scheme indicating that the called service uses Stork to locate and
+ * select the service instance. The `my-service` part is the service name. This is used to configure Stork discovery
+ * and selection in the `application.properties` file.
+ */
+@RegisterRestClient(baseUri = "stork://my-service")
+public interface MyService {
+
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    String get();
+}
+----
+
+It's a straightforward REST client interface containing a single method. However, note the `baseUri` attribute.
+It starts with `stork://`.
+It instructs the REST client to delegate the discovery and selection of the service instances to Stork.
+Notice the `my-service` part in the URL.
+It is the service name we will be using in the application configuration.
+
+It does not change how the REST client is used.
+Create the `src/main/java/org/acme/FrontendApi.java` file with the following content:
+
+[source, java]
+----
+package org.acme;
+
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+/**
+ * A frontend API using our REST Client (which uses Stork to locate and select the service instance on each call).
+ */
+@Path("/api")
+public class FrontendApi {
+
+    @RestClient MyService service;
+
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    public String invoke() {
+        return service.get();
+    }
+
+}
+----
+
+It injects and uses the REST client as usual.
+
+== Stork configuration
+
+Now we need to configure Stork for using Kubernetes to discover the red and blue instances of the service.
+
+In the `src/main/resources/application.properties`, add:
+
+[source, properties]
+----
+quarkus.stork.my-service.service-discovery.type=kubernetes
+quarkus.stork.my-service.service-discovery.k8s-namespace=development
+quarkus.stork.my-service.service-discovery.application=color-service
+quarkus.stork.my-service.load-balancer.type=random
+----
+
+`stork.my-service.service-discovery` indicates which type of service discovery we will be using to locate the `my-service` service.
+In our case, it's `kubernetes`.
+If you are connected to a kubernetes cluster, you don't need to configure the access to it. Otherwise, set the proper Kubernetes url using the `quarkus.stork.my-service.service-discovery.k8s-host` property.
```

Comment:
```suggestion
If you are connected to a Kubernetes cluster, you don't need to configure the access to it. Otherwise, set the proper Kubernetes url using the `quarkus.stork.my-service.service-discovery.k8s-host` property.
```